### DISCUSSION TITLE
$ git merge master 'already up-to-date' after changes made

### DISCUSSION THREAD

**Discussion by Strongeyed** [swellstores/community#206] [Help] [answered]
I've made changes to my products but my website does not publish after using $ git push, the merge master also gives 'everything up-to-date'.

**Comment by ericingram** [comment, marked as answer]
Have you tried changing the version in the repository before doing a push? Like this:

`npm version patch`

**Reply by Strongeyed** [reply]
That actually worked @ericingram however one error showing, not sure how this affects the storefront?

<img width="752" alt="Screenshot 2023-04-11 at 9 57 29 PM" src="https://user-images.githubusercontent.com/128098520/231126660-3a7d9d56-2a04-4875-b397-5894b21048f6.png">

**Reply by ericingram** [reply]
Thanks for confirming, looking into that error

**Reply by Strongeyed** [reply]
Any news on the error @ericingram still getting this when publishing

**Reply by ericingram** [reply]
Did you get it working after all?

**Reply by Strongeyed** [reply]
Yes!😉 still getting the error above but it published in the end which is great
